map2csv/app/streamlit_app.py

```
@@ -0,0 +1,185 @@
+    if "clicked_points" not in st.session_state:
+        st.session_state["clicked_points"] = []
+    if "numbers" not in st.session_state:
+        st.session_state["numbers"] = []
+
+    if "homography" not in st.session_state:
+        st.write("## 位置合わせ")
+        st.write("画像上の2点以上をクリックして、対応する緯度経度を入力してください。")
+        
+        try:
+            font = ImageFont.truetype("/usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf", 20)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)